lms/templates/ccx/schedule.underscore

```diff
@@ -12,37 +17,39 @@
     <tbody>
       <% _.each(chapters, function(chapter) { %>
         <tr class="chapter collapsed" data-location="<%= chapter.location %>" data-depth="1">
-          <td class="unit">
-            <a href="#"><i class="fa fa-caret-right toggle-collapse"></i></a>
-            <%= chapter.display_name %>
+          <td class="unit" aria-label="Link chapter <%= chapter.display_name %>">
+            <a href="#" class="toggle-collapse"><i class="fa fa-caret-right"></i></a>
+            <span aria-label="Section <%= chapter.display_name %>"><%= chapter.display_name %></span>
           </td>
-          <td class="date start-date"><a><%= chapter.start %></a></td>
-          <td class="date due-date"><a><%= chapter.due %></a></td>
-          <td><a href="#" class="remove-unit">
-            <i class="fa fa-remove"></i> <%- gettext('remove') %>
+          <td class="date start-date"><button><%= chapter.start %></button></td>
```

Comment:
If the button text is the start date alone, it no longer describes the action of clicking it (which opens the dialog to change it).  I recommend adding screen reader text to this button that appends that information to the button text e.g. <button><%= chapter.start %><span class="sr"> <%- gettext('Click to change') %></span></button>